.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
Comments aren't a part of the JSON standard (even if VSCode supports them in it for some reason), I'd prefer we didn't include them here so we don't have invalid JSON files floating around.

Reply:
OK, removed the comments.